### PR TITLE
Few corrections in README.

### PR DESCRIPTION
**Pull request by ahmad19**

### CODE REVIEW

**Comment by dblock** [comment]
Thanks. No need for changelog for README updates, you can remove that one pls. Appreciate it.

**Comment by ahmad19** [comment]
@dblock  You're right about not putting the README updates in the CHANGELOG but the @dangerpr-bot does not allow the PR without it. Should i ignore it ?

**Comment by dblock** [comment]
Yes, lets ignore the bot. 

If you feel motivated, https://github.com/dblock/danger-changelog/issues/14 is about making the bot smarter to ignore .md changes.

File: README.md [hunk 1]
```diff
@@ -84,7 +84,7 @@ end
 
 Command match data includes `match['bot']`, `match['command']` and `match['expression']`. The `bot` match always checks against the `SlackRubyBot::Config.user` and `SlackRubyBot::Config.user_id` values obtained when the bot starts.
 
-The `command` method can take strings, which will have be escaped with `Regexp.escape`, and regular expressions.
+The `command` method can take strings, which will have to escaped with `Regexp.escape`, and regular expressions.
```
Comment (dblock) [inline README.md]: to _be_ escaped I think is more correct ;)

Comment (ahmad19) [reply README.md]: yes that make sense :)

**Comment by dblock** [comment]
Thanks! Can I be annoying? Could you please fix the spelling in the commit message? It's _grammar_. Also squash your commits please.

**Pull request merged**